.gitignore

```diff
@@ -1,2 +1,3 @@
 
 cache/index.json
+config/config.ini
```

Comment:
There is only `sample.ini` in the `config` folder. Could you please remove this from the `.gitignore`?